scripts/exgfs_wave_nawips.sh

```diff
@@ -14,8 +14,8 @@
 source "$HOMEgfs/ush/preamble.sh"
 
 #export grids=${grids:-'glo_30m at_10m ep_10m wc_10m ao_9km'} #Interpolated grids
-export grids=${grids:-'glo_10m gso_15m ao_9km'}  #Native grids
-export RUNwave=${RUNwave:-${RUN}${COMPONENT}}
+export grids=${grids:-'glo_30m'}  #Native grids
```

Comment:
Is this not being over-written when what's coming into this routine?

Reply:
It didn't seem to be set upstream, because the default here was being used.